src/com/palmergames/bukkit/towny/object/Resident.java

```diff
@@ -27,38 +27,30 @@
 import java.util.Arrays;
 import java.util.List;
 
-public class Resident extends TownBlockOwner implements ResidentModes, TownyInviteReceiver{
-
+public class Resident extends TownBlockOwner implements ResidentModes, TownyInviteReceiver {
 	private List<Resident> friends = new ArrayList<>();
-	@SuppressWarnings("unused") // Feature is disabled as of MC 1.13, maybe it'll come back.
```

Comment:
Why did you remove this.

Reply:
If it's unused and suppressed we can just remove it and use git history to check how we did it back then, this is how we do it in other cases?

Reply:
The only way I'll remember to readd it someday is if it is still in place.

Reply:
I can comment it out, then it's not a variable and then we don't need a @SuppressWarnings